### DISCUSSION TITLE
Schedule Interval Last Friday of every month

### DISCUSSION THREAD

**Discussion by dipayandev** [apache/airflow#30575] [Q&A]
### Description

Hi Team,

We have a requirement that we need to schedule a Airflow Job on last Friday of every month.

We tried implementing with the below, but it didn't work
We have scheduled it for schedule_interval= `0 00 16 ? * 6L`

But it throws DAG import error:

> Invalid timetable expression: [0 00 16 ? * 6l] is not acceptable 

Could you please help us with the requirement.

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by tanelk** [comment]
Custom timetables is the feature you are looking for:
https://airflow.apache.org/docs/apache-airflow/stable/howto/timetable.html
